### ISSUE TITLE
Auto close html tags

### ISSUE THREAD

**Issue by Dentosal** [atom/language-html#63]
I used to use Sublime text before I changed to Atom. They had a nice feature: When writing html, if I wrote `</`the editor automatically completed it to full closing html tag without. It doesn't required even pressing some key to expand.

Please label this to enhancement.

**Comment by mnquintana** [comment]
@Dentosal I think you were using some community package for that behavior in Sublime - when I type `<div>` in an HTML doc in Sublime it doesn't autoclose. In both Atom and Sublime however you can close open tags with `Cmd/Ctrl-Alt-.`. Nonetheless, this is an interesting enhancement idea.

**Comment by ChrisChiera** [comment]
To add, I am also a Sublime text user. I have no extra plugin for auto complete or auto closing, just the core functionality to my knowledge.

When I type, "<div" and tab it turns it into, "<div></div>" with the cursor in between them automatically. Or If I type "<ul></" it automatically adds, "ul>". Would be great if this was core functionality in Atom or at least ability to enable/disable it via a setting as I would think it's a popular feature. There is currently a plugin that does it that has been downloaded over 61K times <https://atom.io/packages/autoclose-html>. Since Atom is prime to be the next best text editor, I can understand not wanting to add too many core features since it can get bloated and turn into the next Dreamweaver, but hopefully some features like this may make it into core. Just my two cents though!

**Comment by maciejpi** [comment]
@mnquintana this feature was added in Sublime 3.

**Comment by 50l3r** [comment]
+1 The"</" type is more faster than shortcut

**Comment by iSimonWeb** [comment]
:+1:

**Comment by oakley808** [comment]
+1 for "</" closing tags. Love that feature in Sublime 3!

**Comment by blakeperdue** [comment]
+1 would love to be able to type "</" and have the closing tag autocompleted. Hug help for me :)

**Comment by jesse12521** [comment]
+1 for this and other auto closing options. For example in Brackets, typing "<hea" it would display the normal autocomplete options and if you hit enter it would close the opening tag and autocomplete the closing tag, with the cursor left inbetween the tags. Like so: "<header>|</header?". I think having this, and the "</" autocomplete, along with maybe additional closing options would be great core features. 

Definitely need something better than the current shortcut!

**Comment by artitmedia** [comment]
Me too, please !  :+1

**Comment by fredericpfisterer** [comment]
+1 !!!

**Comment by ivanpadavan** [comment]
I think it should be implemented by one of us, guys

**Comment by QuipHop** [comment]
This is still not implemented?
Can't believe...

**Comment by DennHa** [comment]
look at this gem 
https://atom.io/packages/less-than-slash

**Comment by wanavarrete** [comment]
@DennHa Great find bro!

**Comment by rmit-s3383245-James-McKay** [comment]
yes please!

**Comment by HawkonDK** [comment]
Coda for mac have </ as closing, have use it, really miss it, totally the best type of autocomplete i ever have try

**Comment by rdaniels** [comment]
+1

**Comment by ForrestAM** [comment]
I agree, it was something that was really nice and I would really enjoy having it once more
